radix-engine/src/vm/wasm/wasm_validator_config.rs

```diff
@@ -179,31 +181,37 @@ impl Rules for WasmValidatorConfigV1 {
             F64Max => None,
             F64Copysign => None,
             I32WrapI64 => Some(self.weights.i32wrapi64),
-            I32TruncSF32 => None,
-            I32TruncUF32 => None,
-            I32TruncSF64 => None,
-            I32TruncUF64 => None,
-            I64ExtendSI32 => Some(self.weights.i64extendsi32),
-            I64ExtendUI32 => Some(self.weights.i64extendui32),
-            I64TruncSF32 => None,
-            I64TruncUF32 => None,
-            I64TruncSF64 => None,
-            I64TruncUF64 => None,
-            F32ConvertSI32 => None,
-            F32ConvertUI32 => None,
-            F32ConvertSI64 => None,
-            F32ConvertUI64 => None,
+            I32TruncF32S => None,
+            I32TruncF32U => None,
+            I32TruncF64S => None,
+            I32TruncF64U => None,
+            I64ExtendI32S => Some(self.weights.i64extendsi32),
+            I64ExtendI32U => Some(self.weights.i64extendui32),
+            I64TruncF32S => None,
+            I64TruncF32U => None,
+            I64TruncF64S => None,
+            I64TruncF64U => None,
+            F32ConvertI32S => None,
+            F32ConvertI32U => None,
+            F32ConvertI64S => None,
+            F32ConvertI64U => None,
             F32DemoteF64 => None,
-            F64ConvertSI32 => None,
-            F64ConvertUI32 => None,
-            F64ConvertSI64 => None,
-            F64ConvertUI64 => None,
+            F64ConvertI32S => None,
+            F64ConvertI32U => None,
+            F64ConvertI64S => None,
+            F64ConvertI64U => None,
             F64PromoteF32 => None,
             I32ReinterpretF32 => None,
             I64ReinterpretF64 => None,
             F32ReinterpretI32 => None,
             F64ReinterpretI64 => None,
-            SignExt(_) => Some(self.weights.i64extendsi32),
+            // sign-extension instructions
+            I32Extend8S => Some(self.weights.i64extendsi32),
+            I32Extend16S => Some(self.weights.i64extendsi32),
+            I64Extend8S => Some(self.weights.i64extendsi32),
+            I64Extend16S => Some(self.weights.i64extendsi32),
+            I64Extend32S => Some(self.weights.i64extendsi32),
+            _ => todo!("instruction {:?} not covered", instruction),
```

Comment:
Can we list all the instructions that's not covered and promote the `TODO` to `FIXME`?

Reply:
Sure. But be aware that there are 541 variants at the moment. We support ~177 right now. So there will be plenty of TODOs 😄